Tests/ImagineEngineTests/LabelTests.swift

```diff
@@ -122,4 +122,32 @@ final class LabelTests: XCTestCase {
         XCTAssertEqual(labelBClickCount, 1)
         XCTAssertEqual(clickedLabels, [labelB, labelA, labelA])
     }
+
+    func testScaling() {
+        let label = Label(text: "Hello world")
+        let originalSize = label.size
+        let originalFontSize = label.layer.fontSize
+
+        let upscaleFactor: Metric = 2.5
+        let downscaleFactor: Metric = 0.4
+
+        // Upscale the label
+        label.scale = upscaleFactor
+
+        XCTAssertEqual(label.layer.fontSize, originalFontSize * upscaleFactor)
+        XCTAssertGreaterThan(label.size.height, originalSize.height)
+        XCTAssertGreaterThan(label.size.width, originalSize.width)
+
+        // Downscale the label
+        label.scale = downscaleFactor
```

Comment:
Awesome that you're scaling up and down and asserting the result 👍

Reply:
Thanks!